src/components/Upload.vue

```diff
@@ -43,18 +43,30 @@
         </p>
       </div>
     </div>
-    <div class="card blue-grey darken-1" v-show="parseError">
-      <div class="card-content red white-text">
-        <span class="card-title">Could not parse your model</span>
-        <p>We're sorry, we encountered an issue while trying to parse your model.</p>
-        <p v-show="parseErrorMessage"><em>{{ parseErrorMessage }}</em></p>
-        <p>
+
+    <div v-if="parseError">
+      <h4>{{ parseError.title }}</h4>
+      <div class="row">
+        <p class="col s8 offset-s2">
+          We're sorry, we encountered an issue while trying to parse your model. Look below for a detailed error report.
```

Comment:
Please remove the below sentence

> If this is an SBML model, try submitting it to <a href="http://sbml.org/Facilities/Validator/">the SBML Validator</a> to discover potential issues.

since you are now showing the full output just as the validator would.

Reply:
Don't you think it's a good idea to still reference it? The validator seems to output a bit more detailed messages. Compare the above messages from memote.io with this, for the same model:

![image](https://user-images.githubusercontent.com/64745/53509980-3988b000-3abd-11e9-9d58-2922091073f2.png)

Reply:
I believe we have all of that information available as well but it's possibly not being aggregated in full. Alright, let's still reference it for the moment then.